src/views/assets/Cloud/Account/components/CreateDialog.vue

```diff
@@ -118,10 +118,6 @@ export default {
     &.is-process {
       color: var(--color-primary);
 
-      .el-step__line {
-        background-color: var(--color-primary);
-      }
-
       .el-step__icon {
         border-color: var(--color-primary);
       }
```

Comment:
As of January 2021, these lines were not included in the provided code snippet. Please provide an active copy of the problematic line(s), which should be:

```
&.is-process {
    color: var(--color-primary);
    .el-step__line {
        background-color: var(--color-primary);
    }
}

@export default;
``` 

I hope this helps! Let's continue with your next question or request.